frontend/src/components/common/Filter/components/FilterRange/style.ts

```diff
@@ -38,6 +38,9 @@ export const useSliderStyles = makeStyles({
       height: 14,
       marginLeft: -7,
       width: 14,
+      "&:focus": {
```

Comment:
Ah Science Design System is currently having a [discussion](https://czi-sci.slack.com/archives/C032S43KKFV/p1646681775446979?thread_ts=1646438838.295509&cid=C032S43KKFV) around this now as well!

In short, for accessibility, if we don't like the default outlines we'll need to mitigate it instead of just removing it. The link below provides three alternatives that may be worth discussing with the PM and design about?

https://www.a11yproject.com/posts/never-remove-css-outlines/

<img width="854" alt="Screen Shot 2022-03-07 at 11 34 10 AM" src="https://user-images.githubusercontent.com/6309723/157151569-6734a232-7d87-4c47-b2da-01a785a21eec.png">

Reply:
Ok, great thanks @tihuan, I'll ask for direction from the team!

Reply:
Thanks @tihuan! I agree with Maya that I would only expect the focus state styling for keyboard navigation. We have hover states and active states for mouse events already. Those aren't the exact same as a focus state but I'm not sure what need a focus state would address that hover and active states don't already. So, I'd vote in favor of option 3. @frano-m